src/engine/prague.md

```diff
@@ -108,9 +48,13 @@ This method follows the same specification as [`engine_newPayloadV3`](./cancun.m
 
 1. Client software **MUST** return `-38005: Unsupported fork` error if the `timestamp` of the payload does not fall within the time frame of the Prague fork.
 
+2. Client software **MUST** incorporate `executionRequestsHash` into the `blockHash` validation process.
+That is, if `executionRequestsHash` does not match the execution requests commitment in the execution layer block header
```

Comment:
https://eips.ethereum.org/EIPS/eip-7685 is heavily outdated now. It would be good to have it updated so we can refer to the actual name of the field in the execution block header where we store the commitment. Not a blocker for this PR but a nice addition later on.

Reply:
Resolved by https://github.com/ethereum/EIPs/pull/8916